config/checkstyle_resources_suppressions.xml

```diff
@@ -66,4 +66,12 @@
   <suppress checks="OuterTypeFilename"
     files="src[\\/]it[\\/]resources[\\/].*[\\/]SuppressionXpathRegressionOuterTypeFilename.*\.java"/>
 
+  <!-- intentional problem for testing -->
+  <suppress checks="RegexpSingleline"
+    files="[\\/]InputJavadocContentLocationTrailingSpace\.java"/>
+  <suppress checks="RegexpSingleline"
+    files="[\\/]InputJavadocParagraphCorrect\.java"/>
+  <suppress checks="RegexpSingleline"
+    files="[\\/]InputJavadocTagContinuationIndentationGuavaFalsePositive\.java"/>
```

Comment:
You confirmed that there weren't any trailing whitesspaces we could remove? Do any files have more than 1 and maybe pitest will allow us to leave just 1 of them?

Reply:
I've removed all the whitespaces that were possible.